src/extra/python/isca/templates/compile.sh

```diff
@@ -15,12 +19,8 @@ template_debug={{ template_dir }}/mkmf.template.debug
 execdir={{ execdir }}        # where code is compiled and executable is created
 executable={{ executable_name }}
 
-netcdf_flags=`nc-config --fflags --flibs`
+netcdf_flags=`nf-config --fflags --flibs`
```

Comment:
Given that this seems to vary wildly between machines, it might be worth pulling it out into the environment file